modules/core/include/opencv2/core/mat.hpp

```diff
@@ -2408,7 +2411,7 @@ class CV_EXPORTS UMat
     //! assignment operators
     UMat& operator = (const UMat& m);
 
-    Mat getMat(int flags) const;
+    CV_WRAP_AS(get) Mat getMat(int flags = ACCESS_RW) const;
```

Comment:
Result lifetime of this method must be restricted to local code scope.

Nearest Python alternative is "with" statement (enter / exit) methods.